scaling_performance/using_cpu_manager.adoc

```diff
@@ -0,0 +1,86 @@
+[[scaling-performance-using-cpu-manager]]
+= Using CPU Manager
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[what-cpu-manager-does]]
+== What CPU Manager Does
+
+CPU Manager manages groups of CPUs and constrains workloads to specific CPUs.
```

Comment:
@jeremyeder What are some specific use cases for CPU manager? When is it used and why do users care?

Reply:
This feature is useful for workloads that have some of these attributes:

* Require as much CPU time as possible (eliminate jitter).
* Are sensitive to processor cache misses.
* Are low-latency network applications.
* Coordinate with other processes and benefit from sharing a single processor cache.